### ISSUE TITLE
postcss.plugins was deprecated

### ISSUE THREAD

**Issue by agadacz-cognite** [dbtedman/postcss-prefixwrap#104]
PostCSS had moved to version 8.0 recently and therefore using postcss-prefixwrap shows the following message:

```
postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
```

This is an issue for tests which fail whenever any non PASS message appears at the summary, what in turn creates confusion in Jenkins having test suite fail while all the tests are green.

**Comment by dbtedman** [comment]
Thanks @agadacz-cognite for reporting this issue.

I should be able to implement a fix for this issue that will allow support for both new versions of postcss and for older ones. I will reply on this thread when a fix is available.

**Comment by agadacz-cognite** [comment]
Thank you!

**Comment by dbtedman** [comment]
Hi @agadacz-cognite a fix has been prepared, It will be available in NPM as version 1.18.0 soon
